extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/Args.scala

```diff
@@ -19,10 +19,6 @@ case class Args(
     "Path to directory where the extracted raw ENCODE metadata should be written"
   )
   outputDir: String,
-  @HelpMessage(
-    "Batch size that defines the number of elements in a batch when making certain API calls"
-  )
-  batchSize: Long,
```

Comment:
It doesn't really make sense to parameterize this here; there's a size that works well in prod, so it's better to hard-code it and avoid confusion.